### PR TITLE
fix: correct host / config / custom image

### PR DESCRIPTION
**Pull request by vpol**

### CODE REVIEW

File: .infra/clickhouse-sync.yml [hunk 1]
```diff
@@ -58,10 +58,10 @@ connector.class: "io.debezium.connector.postgresql.PostgresConnector"
 offset.storage: "io.debezium.storage.jdbc.offset.JdbcOffsetBackingStore"
 
 # offset.storage.jdbc.offset.table.name: The name of the database table where connector offsets are to be stored.
-offset.storage.jdbc.offset.table.name: "altinity_sink_connector.replica_source_info"
+offset.storage.jdbc.offset.table.name: "api.replica_source_info"
 
 # offset.storage.jdbc.url: The JDBC URL for the database where connector offsets are to be stored.
-offset.storage.jdbc.url: "jdbc:clickhouse://%clickhouse_host%:%clickhouse_port%/altinity_sink_connector"
```
Comment (omBratteng) [inline .infra/clickhouse-sync.yml:64]: Just sanity check, does the updated secret include the `//`?

Comment (vpol) [reply .infra/clickhouse-sync.yml:64]: yes, it now starts with https:// 🙈

Comment (omBratteng) [reply .infra/clickhouse-sync.yml:64]: 👍 just had to check that it was intended

Comment (omBratteng) [reply .infra/clickhouse-sync.yml:64]: Also just checking the change of `altinity_sink_connector` to `api` was intended. In altinity_sink_connector` it creates the table `replica_source_info`

Comment (vpol) [reply .infra/clickhouse-sync.yml:64]: Yes, everything is intended. I've tested this config in a different project.

Comment (omBratteng) [reply .infra/clickhouse-sync.yml:64]: Purrfect 👍

**Review by omBratteng — APPROVED** [review]

**Pull request merged**